src/Apps/Artist/Routes/Overview/ArtistOverviewRoute.tsx

```diff
@@ -98,6 +65,53 @@ export const ArtistOverviewRouteFragmentContainer = createFragmentContainer(
   {
     artist: graphql`
       fragment ArtistOverviewRoute_artist on Artist {
+        filterArtworksConnection(sort: "-weighted_iconicity", first: 10) {
+          edges {
+            node {
+              ...ShelfArtwork_artwork
+              internalID
+              slug
+            }
+          }
+        }
+        insights {
+          entities
+          description
+          label
+          kind
+        }
+        artistSeriesConnection(first: 50) {
+          edges {
+            node {
+              internalID
+              ...ArtistSeriesItem_artistSeries
+            }
+          }
+        }
+        articlesConnection(
+          first: 10
+          sort: PUBLISHED_AT_DESC
+          inEditorialFeed: true
+        ) {
+          edges {
+            node {
+              ...CellArticle_article
+              internalID
+              slug
+              title
+            }
+          }
+        }
+        showsConnection(first: 5, sort: END_AT_ASC, status: "running") {
+          edges {
+            node {
+              ...CellShow_show
+              internalID
+              slug
+              href
+            }
+          }
+        }
```

Comment:
OK, so let's talk about the necessity of doing this. We have an empty state that we want to display; what's the minimal amount of information that can be used to display it? Here we're fetching _alot_ of data, and then using nearly none of it.

For instance, any connections you can just query the counts and completely ignore the data:

```graphql
artistSeriesConnection(first: 0) {
  totalCount
}
```

Where as now you're fetching the first 50 and including a lot of data and unused fragments.

Etc.

Reply:
yep I was working against the clock to do QA and wound up over-fetching data to render the empty state. Good catch, going to take care of this now.